pkg/model/components/cilium.go

```diff
@@ -39,7 +39,7 @@ func (b *CiliumOptionsBuilder) BuildOptions(o interface{}) error {
 	}
 
 	if c.Version == "" {
-		c.Version = "v1.8.4"
+		c.Version = "v1.9.3"
```

Comment:
Maybe now is a good time to use latest version?
```suggestion
		c.Version = "v1.9.4"
```